lib/streamlit/elements/widgets/button.py

```diff
@@ -925,9 +1051,6 @@ def _page_link(
         if help is not None:
             page_link_proto.help = dedent(help)
 
-        if use_container_width is not None:
-            page_link_proto.use_container_width = use_container_width
```

Comment:
Related note: I think it also makes sense to mark the `use_container_width` property in the proto as deprecated with a comment to use the new width config.